### DISCUSSION TITLE
require() of ES module not supported

### DISCUSSION THREAD

**Discussion by jonasfeuss** [gatsbyjs/gatsby#37472] [Help]
I have a problem with my Gatsby website. Every time I want to deploy my website I get this error code:

require() of ES Module /usr/src/app/www/node_modules/package-json/node_modules/lowercase-keys/index.js from /usr/src/app/www/node_modules/package-json/node_modules/cacheable-request/src/index.js not supported.

Instead change the require of /usr/src/app/www/node_modules/package-json/node_modules/lowercase-keys/index.js in /usr/src/app/www/node_modules/package-json/node_modules/cacheable-request/src/index.js to a dynamic import() which is available in all CommonJS modules.

What can I do to fix this error?

**Comment by LekoArts** [comment]
It seems like you're using a version of either `package-json` and/or `lowercase-keys` that is ESM-only. You could also try using the new ESM feature, maybe that helps with your usage (https://www.gatsbyjs.com/docs/reference/release-notes/v5.3/#es-modules-esm-in-gatsby-files). But without more details I'm afraid I can't help.

**Reply by jonasfeuss** [reply]
This is my gatsby-config file. Does this help you or what exactly do you need to help me with the problem?

require(`dotenv`).config()

const shouldAnalyseBundle = process.env.ANALYSE_BUNDLE

module.exports = {
  siteMetadata: {
    // You can overwrite values here that are used for the SEO component
    // You can also add new values here to query them like usual
    // See all options: https://github.com/LekoArts/gatsby-themes/blob/main/themes/gatsby-theme-cara/gatsby-config.js
    siteTitle: `Jonas Feuß`,
    siteTitleAlt: `Jonas Feuß`,
    siteHeadline: `Jonas Feuß`,
    siteUrl: `https://jonasfeuss.de`,
    siteDescription: `Jonas Feuß - Webdesigner, Softwareentwickler`,
    siteLanguage: `de`,
    siteImage: `/banner.jpg`,
    author: `@jonasfeuss`,
  },
  plugins: [
    {
      resolve: `@lekoarts/gatsby-theme-cara`,
      // See the theme's README for all available options
      options: {},
    },
    {
      resolve: `gatsby-plugin-manifest`,
      options: {
        name: `Cara - @lekoarts/gatsby-theme-cara`,
        short_name: `Cara`,
        description: `Playful and Colorful One-Page portfolio featuring Parallax effects and animations`,
        start_url: `/`,
        background_color: `#141821`,
        // This will impact how browsers show your PWA/website
        // https://css-tricks.com/meta-theme-color-and-trickery/
        // theme_color: `#f6ad55`,
        display: `standalone`,
        icons: [
          {
            src: `/android-chrome-192x192.png`,
            sizes: `192x192`,
            type: `image/png`,
          },
          {
            src: `/android-chrome-512x512.png`,
            sizes: `512x512`,
            type: `image/png`,
          },
        ],
      },
    },
    `gatsby-plugin-gatsby-cloud`,
    shouldAnalyseBundle && {
      resolve: `gatsby-plugin-webpack-bundle-analyser-v2`,
      options: {
        analyzerMode: `static`,
        reportFilename: `_bundle.html`,
        openAnalyzer: false,
      },
    },
  ].filter(Boolean),
}
